Library/Formula/nccmp.rb

```diff
@@ -0,0 +1,24 @@
+class Nccmp < Formula
+  desc "Compare two NetCDF files bitwise, semantically or with a tolerance."
+  homepage "http://nccmp.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/nccmp/nccmp-1.7.4.1.tar.gz"
+  sha256 "985b50505dd7517710c5699cb9ee1028c6d75b06a17649bb911d838c4908dc69"
+
+  depends_on "netcdf"
```

Comment:
Ah. You can't have mandatory cross-tap dependencies in the core. This will either need to be:
```ruby
depends_on "homebrew/science/netcdf" => :optional
```
Or this formula will need submitting to the Science tap.